components/inputs/input-text.js

```diff
@@ -469,11 +469,12 @@ class InputText extends FocusMixin(LabelledMixin(FormElementMixin(SkeletonMixin(
 	}
 
 	_getAriaLabel() {
-		if (this.label && (this.labelHidden || this.labelledBy)) {
-			return this.label;
-		}
-		if (this.hasAttribute('aria-label')) {
-			return this.getAttribute('aria-label');
+		let label;
+		if (this.label && (this.labelHidden || this.labelledBy)) label = this.label;
+		if (this.hasAttribute('aria-label')) label = this.getAttribute('aria-label');
```

Comment:
Previously, if `aria-label` AND `label` were set, `label` would take precedence. Now `aria-label` will overwrite it.

```suggestion
		let label;
		if (this.label && (this.labelHidden || this.labelledBy)) {
			label = this.label;
		} else if (this.hasAttribute('aria-label')) {
			label = this.getAttribute('aria-label');
		}
```